azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -710,6 +714,7 @@ V1ObjectMeta createPodMetadata(final int executionId, Map<String, String> flowPa
   String getFlowContainerCPURequest(final Map<String, String> flowParam) {
     if (flowParam == null || flowParam.isEmpty() || !flowParam
         .containsKey(FlowParameters.FLOW_PARAM_FLOW_CONTAINER_CPU_REQUEST)) {
+      this.cpuLimit = this.cpuLimitByConf;
```

Comment:
for my understanding, why do we have variables cpuLimit, memorylimit or any resourcelimits as instance variables? I was wondering about the implication of this because we use this `KubernatesContainerizedImpl` instance to create pods for all the executions. Please correct me if I am missing something obvious!

Reply:
Thanks for the suggestion. I'm eliminating all cpu and memory limit instances in the class.